### PR TITLE
Add 60s waiting to fixing the issues, when hooks pods get into "running" late

### PR DESCRIPTION
**Pull request by weinliu**
Failed log:
`
waiting for operation up to 3600 seconds..
    When I get project pod                                                              # features/step_definitions/project.rb:255
      [07:24:05] INFO> Shell Commands: oc get pod --config=/home/weinliu/workdir/vm251-93-weinliu/ocp4_testuser-0.kubeconfig -n 6jhvt
      NAME             READY   STATUS              RESTARTS   AGE
      hooks-1-wrcsm    1/1     Terminating         0          15s
      hooks-2-deploy   0/1     ContainerCreating   0          9s
      [07:24:07] INFO> Exit Status: 0
    Then the output should match:                                                       # features/step_definitions/common.rb:33
      | hooks-2.*Running |
`
The changes fix this

### CODE REVIEW

**Comment by weinliu** [comment]
@jhou1 @sunilcio @lyman9966  Guys, could you help to take a look?

File: features/cli/deploy.feature [hunk 1]
```diff
@@ -676,9 +676,14 @@ Feature: deployment related features
     Given I wait until number of replicas match "0" for replicationController "hooks-1"
     Then I wait for the "hooks-2" rc to appear
     And I wait until number of replicas match "1" for replicationController "hooks-2"
-    When I get project pod
-    Then the output should match:
-      | hooks-2.*Running |
+    Given I wait up to 60 seconds for the steps to pass:
+      """
```
Comment (jhou1) [inline features/cli/deploy.feature]: Can you please remove the indent to align the `"""` with the above line. There is no need to indent here.

File: features/cli/deploy.feature [hunk 2]
```diff
@@ -676,9 +676,14 @@ Feature: deployment related features
...
+      When I run the :get client command with:
+       | resource | po   |
+      Then the step should succeed
+      And the output should match:
+       | hooks-2.*Running|
```
Comment (jhou1) [inline features/cli/deploy.feature]: Add a space between the later to the `| hooks-2.*Running |`

**Comment by jhou1** [comment]
A small suggestion to the formatting. Looks good.

File: features/cli/deploy.feature (hunk 1 above)
Comment (weinliu) [reply features/cli/deploy.feature]: Updated

File: features/cli/deploy.feature (hunk 2 above)
Comment (weinliu) [reply features/cli/deploy.feature]: Thanks, updated.

**Comment by pruan-rht** [comment]
@weinliu you know what, I'll merge this and add the `set_env` change later.  Thanks!

**Comment by weinliu** [comment]
> @weinliu you know what, I'll merge this and add the `set_env` change later. Thanks!

cool, thanks! I was about to do it today

**Pull request merged**